include/inexor/vulkan-renderer/wrapper/window.hpp

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
+#include <spdlog/spdlog.h>
```

Comment:
`spdlog/spdlog.h` doesn't need to be included in the header.

Reply:
Right. I'll move it to the `.cpp`.

Reply:
Done.